src/components/home/A320Header.tsx

```diff
@@ -20,19 +20,21 @@ export const A320Header = () => {
                 muted
                 loop
             />
-            <Container className="relative max-w-6xl min-h-screen">
+            <Container className="relative min-h-screen">
                 <div className="absolute bottom-0 left-0 h-2/3 px-page">
-                    <h1 className="text-8xl font-bold">
-                        <span className="text-teal">A32</span>
-                        <span className="text-blue">N</span>
-                        X
+                    <h3 className="text-3xl font-semibold text-cyan-dark">
```

Comment:
```suggestion
                    <h3 className="text-3xl font-semibold text-primary">
```

We should look into named colours. See [here](https://tailwindcss.com/docs/customizing-colors#naming-your-colors)